code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -60,4 +61,6 @@
 /mob/living/carbon/human/Process_Spacemove(movement_dir = 0) //Temporary laziness thing. Will change to handles by species reee.
 	if(..())
 		return 1
-	return dna.species.space_move()
+	var/mob/living/carbon/human/H = src
+	if(ishuman(H))

```

Comment:
totally unnecessary, simply by the type you know it is a human
you probably want to check if(src) instead?